examples/heart_rate_collector/main.c

```diff
@@ -895,27 +895,49 @@ static void ble_evt_dispatch(adapter_t * adapter, ble_evt_t * p_ble_evt)
 /**@brief Function for application main entry.
  *
  * @param[in] argc Number of arguments (program expects 0 or 1 arguments).
- * @param[in] argv The serial port of the target nRF5 device (Optional).
+ * @param[in] argv The serial port and baud rate of the target nRF5 device (Optional).
  */
 int main(int argc, char * argv[])
 {
     uint32_t error_code;
     char *   serial_port;
+    uint32_t baud_rate;
```

Comment:
Initialise variables to default when declaring, so you can also avoid _else_ branches.